lib/streamlit/components/v1/custom_component.py

```diff
@@ -221,7 +221,7 @@ def deserialize_component(ui_value, widget_id=""):
 
         # We currently only support writing to st._main, but this will change
         # when we settle on an improved API in a post-layout world.
-        dg = _main
+        dg = main_dg
```

Comment:
Is `_main` and `main_dg` the same?

Reply:
yeah it's imported [here](https://github.com/streamlit/streamlit/blob/develop/lib/streamlit/__init__.py#L69) in the `lib/streamlit/__init__.py` and assigned [here](https://github.com/streamlit/streamlit/blob/develop/lib/streamlit/__init__.py#L136) to `_main`. I figured the more narrow import might be nicer